components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/store/JDBCRecoveryDataStore.java

```diff
@@ -88,13 +106,19 @@ public void store(UserRecoveryData recoveryDataDO) throws IdentityRecoveryExcept
     }
 
     @Override
-    public UserRecoveryData load(User user, Enum recoveryScenario, Enum recoveryStep, String code)
-            throws IdentityRecoveryException {
+    public UserRecoveryData load(User user, Enum recoveryScenario, Enum recoveryStep, String code) throws IdentityRecoveryException {
```

Comment:
please check line size

Reply:
Thanks! Fixed in 16be4b52a13a2587d7637024d265d0f3e6c61d27